src/common/Random.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by Antti Hyvarinen on 07.05.22.
+//
```

Comment:
Use new license header :)

Reply:
Technically it's not my code, it's probably from MiniSat.  I'm not sure what to say there...  Should I put Minisat's license and Een & Sorenson as authors, addition to me?